.circleci/config.yml

```diff
@@ -888,7 +888,7 @@ jobs:
                   path: ~/transformers/test_preparation.txt
             - run: |
                   if [ -f test_list.txt ]; then
-                    python -m pytest -n 1 --max-worker-restart=0 --dist=loadfile -s --make-reports=tests_onnx $(cat test_list.txt) -k onnx | tee tests_output.txt
+                    python -m pytest -n 1 --max-worker-restart=0 --dist=loadfile -rA -s --make-reports=tests_onnx $(cat test_list.txt) -k onnx | tee tests_output.txt
```

Comment:
I'm not sure if this change will interfere with our internal Slack reporting. Gently pinging @ydshieh for his input 🙏

Reply:
When using `--make-reports`, there is no more need to use `-rA`.

See [here](https://github.com/huggingface/transformers/blob/6faf283288ce3390281ad8c1d37ccb13f2d03990/src/transformers/testing_utils.py#L1283)

I believe `-rA` will display the very detailed results on the terminal, which we would like to avoid. Instead, the summary is saved to files when we use `--make-reports` (this is not a pytest flag, but our own custom flag).

see [pytest doc](https://docs.pytest.org/en/7.1.x/how-to/output.html#producing-a-detailed-summary-report)

Reply:
@ydshieh I added `-rA` because even though the test was included in `tests_fetcher.py`, `require_tf` actually skipped the test since originally TensorFlow was not installed for `run_tests_onnxruntime`. I wouldn't have known that I needed to add `tf` for `run_tests_onnxruntime` without using `-rA` and checking that the tests I expected to run did run. The artifacts did not say exactly which tests were run, but what would be collected. Does `--make-reports` generate other reports that are not in CircleCI?

Reply:
@rachthree In `ARTIFACTS` tab of this [run_tests_onnxruntime run page](https://app.circleci.com/pipelines/github/huggingface/transformers/46219/workflows/f5b23910-e483-41c6-b313-054d23a620f5/jobs/542677/artifacts), you could find `~/transformers/tests_output.txt`. Click it, at the end, you will see something like
```bash
...
...
SKIPPED [1] tests/onnx/test_onnx.py:86: test is slow
SKIPPED [1] tests/onnx/test_onnx.py:75: test is slow
SKIPPED [400] ../.pyenv/versions/3.7.12/lib/python3.7/unittest/case.py:93: test is slow
SKIPPED [2] tests/onnx/test_onnx_v2.py:370: test is slow
================ 45 passed, 444 skipped, 45 warnings in 46.63s =================
```

In previous runs where TF was not installed, we should be able to see those ONNX tests were skipped for TF.

Reply:
@ydshieh I went back to the CI run for commit [8da5990](https://github.com/huggingface/transformers/pull/18615/commits/8da5990fda344f7eb754bc8a9a0f44fac56b00fd) which didn't have TF and `-rA`, and even though [~/transformers/tests_output.txt](https://output.circle-artifacts.com/output/job/7f7e32ea-de63-485a-ac3a-2acccef11190/artifacts/0/~/transformers/tests_output.txt) didn't show the breakdown, [~/transformers/reports/tests_onnx/summary_short.txt](https://output.circle-artifacts.com/output/job/7f7e32ea-de63-485a-ac3a-2acccef11190/artifacts/0/~/transformers/reports/tests_onnx/summary_short.txt) did. Somehow I missed this when going through the artifacts, my apologies! `-rA` has been removed in [d8f3804](https://github.com/huggingface/transformers/pull/18615/commits/d8f38043867488ed61ed9d62175159acbcc09d1b). Thanks :)

There are other runs in `.circleci/config.yaml` that still have `-rA` from before... will the flag be removed from those runs in the future?

Reply:
@rachthree  Thank you for double checking in commit 8da5990.

 @stas00 @LysandreJik Is there any reason to keep (or not) the flag `-rA` when we already use `--make-reports`?

Reply:
It has been a long time since I wrote that code, does it still work with `-rA`? If so, there is no reason, but please remember that `--make-reports` is a hack, so it's quite possible that that flag was needed.

Please try w/o it and if it still produces the same output, then we may choose to remove it.

It's also possible that it is there if we wanted both - the normal log and the `--make-reports` individual logs.

Reply:
@stas00 For `run_tests_onnxruntime`, it still works. `~/transformers/reports/tests_onnx/summary_short.txt` always has the breakdown, so `-rA` is not needed as long as people know which artifact to look for. However, the output changes for the terminal output  in CircleCI and `~/transformers/tests_outputs.txt` as the short test summary info doesn't show without it. 

As for the rest of the tests in the CircleCI config, I think removing `-rA` would be outside the scope of this PR, but I can try it here or another branch if you'd like. Not having consistency confused me when I looked at the logs of the other tests.

Reply:
@rachthree I think we can merge this PR first, with `-rA` if you prefer. As other tests also have this flag, I won't worry about having it.

We can check if there is a need to remove this flag in a separate PR, but this is not a priority.

Reply:
Please do what's the best. I was just mentioning that if I remembered correctly some of those flags were still needed for `--make-reports` to work as intended.

Reply:
PR's been merged without the `-rA` flag, maintaining previous logging behavior. Thank you all for your review 😊